Modules/posixmodule.c

```diff
@@ -5278,6 +5278,16 @@ parse_posix_spawn_flags(PyObject *setpgroup, int resetids, PyObject *setsigmask,
 #endif
     }
 
+    if (usevfork) {
+#ifdef POSIX_SPAWN_USEVFORK
+        all_flags |= POSIX_SPAWN_SETSCHEDULER;
+#else
+        PyErr_SetString(PyExc_NotImplementedError,
+                "The UseVFork option is not supported in this system.");
```

Comment:
The parameter is called "usevfork", the error message should use the same name (not "UseVFork").